recipes/mm/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "mm" %}
+{% set version = "4.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/aivazis/mm/archive/v{{ version }}.tar.gz
+    sha256: bd8c9309965b78b956c27347811f0e1601152dd35f57e75eeccb5481e0190af8
+
+build:
+  number: 0
+  skip: true  # [win or py2k or pyhon_impl == 'pypy']
+
+requirements:
+  build:
+  host:
+    - zip
+    - python
+  run:
```

```suggestion
requirements:
  run:
```
This project isn't a python package; it's just a python script installed in /bin. So it doesn't need a host section.